internal/namespaces/lb/v1/helper_test.go

```diff
@@ -14,3 +16,37 @@ func createLB() core.BeforeFunc {
 func deleteLB() core.AfterFunc {
 	return core.ExecAfterCmd("scw lb lb delete {{ .LB.ID }}")
 }
+
+func setUpCertificateLocally() core.BeforeFunc {
+	return func(ctx *core.BeforeFuncCtx) error {
+		homeDir := ctx.OverrideEnv["HOME"]
+		privateKeyPath := path.Join(homeDir, "private.key")
+		privateCSRPath := path.Join(homeDir, "private.csr")
+		certificatePath := path.Join(homeDir, "certificate.crt")
+
+		// Generate private key and Certificate Signing Request (CSR)
+		cmd := []string{
```

Comment:
This kind of openssl command should be in the long section in the help to be copy and pasted easily